kubernetes/cronjob/build.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2019 LitmusChaos Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cronjob
+
+import (
+	"errors"
+	"fmt"
+
+	jobtemplatespec "github.com/litmuschaos/kube-helper/kubernetes/jobtemplatespec"
+	batchv1beta1 "k8s.io/api/batch/v1beta1"
+)
+
+// Builder is the builder object for CronJob
+type Builder struct {
+	cronjob *CronJob
+	errs    []error
+}
+
+// NewBuilder returns new instance of Builder
+func NewBuilder() *Builder {
+	return &Builder{
+		cronjob: &CronJob{
+			object: &batchv1beta1.CronJob{},
+		},
+	}
+}
+
+// WithName sets the Name field of CronJob with provided value.
+func (b *Builder) WithName(name string) *Builder {
+	if len(name) == 0 {
+		b.errs = append(
+			b.errs,
+			errors.New("Failed to build Job object: missing Job Name"),
+		)
+		return b
+	}
+	b.cronjob.object.Name = name
+	return b
+}
+
+// WithNamespace sets the Namespace field of CronJob with provided value.
+func (b *Builder) WithNamespace(namespace string) *Builder {
+	if len(namespace) == 0 {
+		b.errs = append(
+			b.errs,
+			errors.New("failed to build Job object: missing namespace"),
+		)
+		return b
+	}
+	b.cronjob.object.Namespace = namespace
+	return b
+}
+
+// WithLabels sets the labels field of CronJob with provided value
+func (b *Builder) WithLabels(labels map[string]string) *Builder {
+	if len(labels) == 0 {
+		b.errs = append(
+			b.errs,
+			errors.New("failed to build Job object: missing labels"),
+		)
+		return b
+	}
+
+	if b.cronjob.object.Labels == nil {
+		b.cronjob.object.Labels = map[string]string{}
+	}
+
+	for key, value := range labels {
+		b.cronjob.object.Labels[key] = value
+	}
+	return b
+}
+
+// WithJobTemplateSpecBuilder sets the jobtemplate of this cronjob
+func (b *Builder) WithJobTemplateSpecBuilder(
+	tmplbuilder *jobtemplatespec.Builder,
```

Comment:
Can you check some issue is here

Reply:
I am just getting an error here and that too because it is yet not merged so not able to pick the code from `litmuschaos/kube-helper/` repo
![Screenshot from 2019-11-22 17-00-13](https://user-images.githubusercontent.com/50315249/69422405-98780e80-0d49-11ea-8ec9-a78845ce9f61.png)


error -
```
cannot use *jobtemplatespecObj.Object (type "github.com/litmuschaos/kube-helper/vendor/k8s.io/api/batch/v1beta1".JobTemplateSpec) as type "github.com/Sanjay1611/kube-helper/vendor/k8s.io/api/batch/v1beta1".JobTemplateSpec in assignmentgo
```

Reply:
okay! Just fix the import indentation part